core/modules/xrdsvc/SsiRequest.cc

```diff
@@ -129,25 +129,19 @@ void SsiRequest::execute(XrdSsiRequest& req) {
                 || (ru.db()    != taskMsg->db())
                 || (ru.chunk() != taskMsg->chunkid())) {
                 reportError("Mismatched db/chunk in TaskMsg on resource db=" + ru.db() +
-                            " chunkId=" + std::to_string(ru.chunk()));
+                        " chunkId=" + std::to_string(ru.chunk()));
                 return;
             }
 
-            // Now that the request is decoded (successfully or not), release the
-            // xrootd request buffer. To avoid data races, this must happen before
-            // the task is handed off to another thread for processing, as there is a
-            // reference to this SsiRequest inside the reply channel for the task,
-            // and after the call to BindRequest.
```

Comment:
Why is this comment block no longer relevant? It seems the basic code has not changed to make it irrelevant.

Reply:
Good point. I put it back.